robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAssetManager.java.vm

```diff
@@ -417,16 +420,28 @@ public final class ShadowAssetManager {
     AttributeResource attribute = findAttributeValue(resId, set, styleAttrStyle, defStyleFromAttr, defStyleFromRes, themeStyleSet);
     while (attribute != null && attribute.isStyleReference()) {
       ResName otherAttrName = attribute.getStyleReference();
-
-      attribute = themeStyleSet.getAttrValue(otherAttrName);
-      if (attribute != null) {
-        attribute = new AttributeResource(resourceLoader.getResourceIndex().getResName(resId), attribute.value, attribute.contextPackageName);
+      ResName resName = resourceLoader.getResourceIndex().getResName(resId);
+
+      AttributeResource otherAttr = themeStyleSet.getAttrValue(otherAttrName);
+      if (otherAttr == null) {
+        strictError("no such attr %s in %s while resolving value for %s", attribute.value, themeStyleSet, resName.getFullyQualifiedName());
+        attribute = null;
+      } else {
+        attribute = new AttributeResource(resName, otherAttr.value, otherAttr.contextPackageName);
       }
     }
 
     return attribute;
   }
 
+  private void strictError(String message, Object... args) {
```

Comment:
Totally what I was thinking. Figured it was worth a larger discussion of how to make it configurable and when to transition...